workflow-templates/repo2docker.yml

```diff
@@ -0,0 +1,17 @@
+# Builds the repository using repo2docker
+
+---
+name: repo2docker
+
+on: [push, pull_request]
```

Comment:
From my experience with https://github.com/ome/.github/pull/6, the main question is whether this should also include a default scheduled configuration so that the workflows are executed regularly

Reply:
considering the problem we had in the past few months with some repo e.g. ilastik, I think this is a good idea.
I will add a default configuration